### ISSUE TITLE
Default value for `aggregatorName`

### ISSUE THREAD

**Issue by PierreMarion23** [plotly/react-pivottable#23]
Hi,
In your documentation, you write that the `aggregatorName` key should take as default value the first key in `aggregators`, but in the code (Utilities.js, line 777), the default value is `Count`.
This is annoying when defining custom aggregators, because the `Count` aggregator is not necessarily in the aggregator list, and you don't always want to specify a `aggregatorName` key, thus leading to an error.
So in my opinion, it would be better to have this default value as specified in the documentation :).

Many thanks,
Pierre

**Comment by nicolaskruchten** [comment]
Good point! I'll see when I have time to do this, and I'd happily accept a PR for it :)

**Comment by debikadutt** [comment]
Is there any update on this?
